examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -698,7 +704,13 @@ def main(args):
     )
 
     # Load scheduler and models
-    noise_scheduler = DDPMScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
+    noise_scheduler = DDPMScheduler.from_pretrained(
+        args.pretrained_model_name_or_path,
+        subfolder="scheduler",
+        rescale_betas_zero_snr=args.scale_scheduler,
+        timestep_spacing="trailing" if args.scale_scheduler else None,
```

Comment:
DDPM doesn't support these

Reply:
also not sure whether we want to unconditionally override the training timestep spacing, because the original paper also shows that it works with `leading` timestep spacing.


it's not clear whether the timestep spacing matters at training time.